phys2denoise/tests/test_rvt.py

```diff
@@ -0,0 +1,30 @@
+import peakdet
+import numpy as np
+from phys2denoise.metrics.chest_belt import rvt
+
+
+def read_fake_phys():
```

Comment:
Since this is data to be used in multiple tests, can you convert it to a fixture? See [here](https://github.com/physiopy/phys2bids/blob/f58755939b668729f122212532f1a63d453f25e3/phys2bids/tests/conftest.py#L45-L48) for an example.

Reply:
:+1:

Reply:
This should be done now.